src/algebra/lie/nilpotent.lean

```diff
@@ -78,3 +78,69 @@ begin
   use k, rw ← le_bot_iff at h ⊢,
   exact le_trans (lie_module.derived_series_le_lower_central_series R L k) h,
 end
+
+section nilpotent_algebras
+
+-- TODO Generalise the below to Lie modules if / when we define morphisms, equivs of Lie modules
+-- covering a Lie algebra morphism of (possibly different) Lie algebras.
+
+variables (R : Type u) (L : Type v) (L' : Type w)
+variables [comm_ring R] [lie_ring L] [lie_algebra R L] [lie_ring L'] [lie_algebra R L']
+
+abbreviation lie_algebra.is_nilpotent (R : Type u) (L : Type v)
+  [comm_ring R] [lie_ring L] [lie_algebra R L] : Prop :=
+lie_module.is_nilpotent R L L
+
+variables {R L L'}
+open lie_algebra
+
+lemma lie_ideal.lower_central_series_map_le_lower_central_series (k : ℕ) {f : L →ₗ⁅R⁆ L'} :
+  lie_ideal.map f (lie_module.lower_central_series R L L k) ≤
+  lie_module.lower_central_series R L' L' k :=
```

Comment:
Does it make sense to `open lie_module (lower_central_series)` to make these statements shorter?

Reply:
Yes, thanks!